azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/IAccountActions.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.lib.auth;
+
+import com.microsoft.azure.toolkit.lib.common.action.Action;
+
+public interface IAccountActions {
```

Comment:
why interface here, do we need to implement it later?

Reply:
yes, and it's implemented by `IntellijAccountActionsContributor`
We just make some action ids here and expect ide-s to provide these actions.
it seems better to be `AccountActionsContributor`, but the `IActionsContributor` is in IDE lib.